### DISCUSSION TITLE
[4.0] share your Joomla

### DISCUSSION THREAD

**Discussion by Magnytu2** [joomla/joomla-cms#32436] [General]
I have been playing with joomla for some time! 4. If we create websites it is above all for sharing. I find it unfortunate that there was not a simple reflection and in the body of Joomla for sharing on facebook or twitter including the first intro image as well as the intro text.

http://www.magnytudeweb.com/fnak.com/index.php

**Comment by dgrammatiko** [comment]
@Magnytu2 there are plenty share buttons out there, from plain html to modern js (https://github.com/dgrammatiko/share-element) Joomla doesn't have to provide something, these buttons are very simple to include in pages/articles/modules or template positions

**Comment by Magnytu2** [comment]
Very easy for you. And it's not a button I'm talking about. It's the simple act of copying and pasting an address from a joomla site to facebook (for example). It's up to each galley to retrieve an image and the introductory text. I know you can use extensions. But it should be native in Joomla. As one could speak of a cookie alert module. It makes sense to me that it should be native. Sharing and warning internet users is basic.

**Reply by dgrammatiko** [reply]
> It's the simple act of copying and pasting an address from a joomla site to facebook (for example

That's not how social sharing is working, the title, description and image need to be defined either in `og:` or `twitter:` eg
![Screenshot 2021-02-16 at 16 29 53](https://user-images.githubusercontent.com/3889375/108084277-4de06180-7074-11eb-8fe1-ac585660e297.png)


In short you need one of the many plugins in JED for this

**Reply by Stuartemk** [reply]
Why not implement this natively? Many other platforms (CMS) have already implemented it natively for a long time.

**Comment by chmst** [comment]
Very nice work. But closing Closing here this is not an issue.

**Comment by chmst** [comment]
Better moving to discussions

**Comment by Magnytu2** [comment]
It is a function request. is it not the right section?
